pkg/portal/graphql/query.go

```diff
@@ -1,13 +1,31 @@
 package graphql
 
 import (
-	"github.com/authgear/graphql-go-relay"
+	relay "github.com/authgear/graphql-go-relay"
 	"github.com/graphql-go/graphql"
 
 	"github.com/authgear/authgear-server/pkg/portal/session"
 	"github.com/authgear/authgear-server/pkg/util/graphqlutil"
 )
 
+var peekCollaborationInvitationInput = graphql.NewInputObject(graphql.InputObjectConfig{
+	Name: "PeekCollaborationInvitationInput",
+	Fields: graphql.InputObjectConfigFieldMap{
+		"code": &graphql.InputObjectFieldConfig{
+			Type:        graphql.NewNonNull(graphql.String),
+			Description: "Collaboration Invitation Code",
+		},
+	},
+})
+
+var peekCollaboratorInvitationPayload = graphql.NewObject(graphql.ObjectConfig{
+	Name: "PeekCollaboratorInvitationPayload",
+	Fields: graphql.Fields{
+		"canAccept": &graphql.Field{Type: graphql.NewNonNull(graphql.Boolean)},
+		"appId": &graphql.Field{Type: graphql.NewNonNull(graphql.String)},
```

Comment:
Our convention is `appID` instead of `appId`. You can see this convention in the schema.